### ISSUE TITLE
V1.9.2.10 - Initializing device monitoring (forever)

### ISSUE THREAD

**Issue by Jagermeister56** [nicehash/NiceHashMiner#1705]
On startup 
loading, please wait...
Initializing device monitoring

this basically goes on forever or a very long time.
and other times it a matter of seconds.

**Comment by Jagermeister56** [comment]
removed "run as admin"
seems to have fix it somehow.

**Comment by Jagermeister56** [comment]
and also has trouble actually closing the application, just hangs there.

**Comment by sangdrax8** [comment]
I am having the same issue, but only after trying to upgrade from 1.9.2.10 to 1.9.2.11 and only on my machine with more than 1 GPU (6 in this case)

I successfully upgraded 2 machines with a single GPU from 1.9.2.10 to 1.9.2.11 with no problems.  When I try on my 6 card machine, it freezes and never gets past monitoring.  If I run the 1.9.2.10 version it still works, so I have 1 machine still on the older version until this gets worked out.

**Comment by Vendigo666** [comment]
Same issue. On 1.9.1.11 and 1.9.2.11. On Windows 7. I have 1060 and 2070 in one rig.
Also few times the bar goes next and freez on cross referencing devices IDs.

**Comment by JirkaJawas** [comment]
1.9.2.10 works fine, but have the same issue with 1.9.2.11. Tested on new installation of Windows Server 2016 and Windows 10 Pro. Both completely updated. There are repeating the same lines in log again and again: 
[2019-08-05 01:59:13] [INFO] [NvidiaMonitorManager] NVML HANDLE:140709660935064
[2019-08-05 01:59:13] [INFO] [NvidiaMonitorManager] NVML HANDLE:140709661025888
[2019-08-05 01:59:13] [INFO] [NvidiaMonitorManager] NVML HANDLE:140709661116712
[2019-08-05 01:59:13] [INFO] [NvidiaMonitorManager] NVML HANDLE:140709661207536
[2019-08-05 01:59:13] [ERROR] [NVML] Getting power info failed with message "EXPECTED_PHYSICAL_GPU_HANDLE", disabling power setting
[2019-08-05 01:59:13] [ERROR] [NVML] Getting power info failed with message "EXPECTED_PHYSICAL_GPU_HANDLE", disabling power setting
[2019-08-05 01:59:13] [ERROR] [NVML] Getting power info failed with message "EXPECTED_PHYSICAL_GPU_HANDLE", disabling power setting
[2019-08-05 01:59:13] [ERROR] [NVML] Getting power info failed with message "EXPECTED_PHYSICAL_GPU_HANDLE", disabling power setting
[2019-08-05 01:59:13] [INFO] [NHM.DeviceMonitoring.DeviceMonitorManager] RestartNVIDIAMonitoring END
[2019-08-05 01:59:13] [INFO] [NHM.DeviceMonitoring.DeviceMonitorManager] RestartNVIDIAMonitoring START
[2019-08-05 01:59:13] [INFO] [NHM.DeviceMonitoring.DeviceMonitorManager] Waiting 2000ms before NVIDIA device monitor re-init

**Comment by luc1an24** [comment]
Hi 
Thank you for reporting this bug.
There seems to be a problem with NVAPI.
Could you please run `CreateLogReport.exe` and send me the archived logs?
You can send them to `domen.kirnkrefl@nicehash.com` so no personal data will be public.
We will try our best to fix that issue as soon as possible.
Thank you for your patience.
In the meantime please use one of the older versions - where power/fan data for some gpus doesn't work but at least NHM doesn't freeze.

**Comment by ToxicYeah** [comment]
I am having this issue too. I'd like to know when it's resolved.

**Comment by iambeto** [comment]
I am having this issue as well, it is only on one machine all other machines are working just fine.

**Comment by luc1an24** [comment]
Closing this issue for easier following of the bug fixing information.
Duplicate of #1704
